docs/tutorials/football.md

```diff
@@ -102,5 +102,5 @@ football.createGame(league.kansasCityChiefs, league.oaklandRaiders, 90);
 ```
 
 ```package
-pxt-kickoff=github:microsoft/pxt-kickoff#v0.0.2
+pxt-kickoff=github:microsoft/pxt-kickoff#v0.0.3
```

Comment:
Can this be just pxt-kickoff=github:microsoft/pxt-kickoff  so it points to the latest version?